src/client/extension.ts

```diff
@@ -153,6 +156,7 @@ export async function activate(context: vscode.ExtensionContext) {
     context.subscriptions.push(hepProvider);
 
     context.subscriptions.push(vscode.debug.registerDebugConfigurationProvider('python', new SimpleConfigurationProvider()));
+    activationDeferred.resolve();
```

Comment:
Is there any reason why it would be better to have `activated` set to `false` and then flip it to `true` here after the activation as resolved? I guess that loss of `const`ness would be unfortunate (basically I'm just trying to learn better TS design 😃 ).

Reply:
It needs to be a promise that can be used from within the unit test. Having a boolean doesn't allow me to **await** on the completion of the above code